src/content/en/fundamentals/architecture/building-components/examples/howto-checkbox.md

```diff
@@ -0,0 +1,617 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-08-14 #}
+{# wf_published_on: 2017-04-06 #}
+{# wf_blink_components: Blink>DOM #}
+
+# HowTo: Components – <howto-checkbox> {: .page-title }
```

Comment:
You cannot have "markdown" in a heading. You can use `HowTo: Components - &lt;howto-checkbox&gt;`. Encoding the entities will STILL FAIL the test, but I've verified it works and I'll fix the tests asap.